topydo/lib/Version.py

```diff
@@ -1,8 +1,8 @@
 """ Version of Topydo. """
 
-VERSION = '0.9'
+VERSION = '0.10-hidden_tag.20160408'
```

Comment:
Perhaps you can rewrite the history a bit such that this change does not appear in history (I see you reverted it in a later commit).